<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="tableLRecord">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    border: 5px solid rgb(22, 17, 58);
	border-radius: 7px;
}

QTableWidget::item:hover {
    background-color: rgb(0, 209, 255);
}

QTableWidget::item:selected {
    background-color: rgb(15, 211, 84) ;

}</string>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*List won't be updated if they lack information specially on Nutrients.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>202</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Meal Global Records</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Here are list of meals available for you to choose from. You may also add meals on the &quot;Modify Records&quot; section.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backEButton">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid rgb(22, 17, 58);
    border-radius: 5px;
    background-color: transparent;
    padding: 2px;
    font-size: 13px;
	color: rgb(22, 17, 58);
}

QPushButton:hover {
    border: 1px solid rgb(22, 17, 58);
    background-color: rgb(233, 231, 249);
}

QPushButton:pressed {
    border: 1px solid rgb(22, 17, 58);
    background-color: rgb(214, 213, 229);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
